### PR TITLE
Fix missed references in previous refactors

### PR DESCRIPTION
**Pull request by fatin99**

### CODE REVIEW

File: docs/diagrams/ArchitectureSequenceDiagram.puml [hunk 1]
```diff
@@ -13,13 +13,13 @@ activate ui UI_COLOR
 ui -[UI_COLOR]> logic : execute("delete 1")
 activate logic LOGIC_COLOR
 
-logic -[LOGIC_COLOR]> model : deletePerson(p)
+logic -[LOGIC_COLOR]> model : deleteStudent(p)
```
Comment (aakanksha-rai) [inline docs/diagrams/ArchitectureSequenceDiagram.puml:16]: This is alright for now but we'll have to change it once we edit the delete student command to delete by matric number instead.

File: docs/diagrams/BetterModelClassDiagram.puml [hunk 2]
```diff
@@ -4,18 +4,18 @@ skinparam arrowThickness 1.1
...
 
-Person *--> Name
-Person *--> Phone
-Person *--> Email
-Person *--> Address
+Student *--> Name
+Student *--> Phone
+Student *--> Email
+Student *--> Matric
 @enduml
```
Comment (aakanksha-rai) [inline docs/diagrams/BetterModelClassDiagram.puml:21]: Looks good!

File: docs/diagrams/DeleteStudentSequenceDiagram.puml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+CommandResult --> DeleteStudentCommand
+deactivate CommandResult
+
+DeleteStudentCommand --> LogicManager : result
+deactivate DeleteStudentCommand
+
+[<--LogicManager
+deactivate LogicManager
+@enduml
```
Comment (aakanksha-rai) [inline docs/diagrams/DeleteStudentSequenceDiagram.puml:69]: Once again, would have to edit it once delete student is updated, but looks good for now! :)

File: docs/diagrams/ModelClassDiagram.puml [hunk 4]
```diff
@@ -32,25 +32,25 @@ Class Tag
...
+Student *--> "*" Tag
 
 Name -[hidden]right-> Phone
-Phone -[hidden]right-> Address
-Address -[hidden]right-> Email
+Phone -[hidden]right-> Matric
+Matric -[hidden]right-> Email
 
-ModelManager -->"1" Person : filtered list
+ModelManager -->"1" Student : filtered list
```
Comment (aakanksha-rai) [inline docs/diagrams/ModelClassDiagram.puml:55]: Will have to update this to include sessions and groups and modules as well. However, I think you don't need to do that part. We can do it in a separate PR. Thanks for refactoring this for us!

File: docs/diagrams/AddStudentSequenceDiagram.puml [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+participant ":Model" as Model MODEL_COLOR
+end box
+
+[-> LogicManager : execute("add n/John Doe p/98765432 e/johnd@example.com m/A0181234G")
+activate LogicManager
+
+LogicManager -> TaTrackerParser : parseCommand("add n/John Doe p/98765432 e/johnd@example.com m/A0181234G")
+activate TaTrackerParser
+
+create AddStudentCommandParser
```
Comment (PotatoCombat) [inline docs/diagrams/AddStudentSequenceDiagram.puml:22]: Just refactoring right? Cos actually a StudentCommandParser is created.

**Review by PotatoCombat — APPROVED** [review]

**Comment by PotatoCombat** [comment]
LGTM!

**Review by aakanksha-rai — COMMENTED** [review]
Looks good! Just need to change that first command. The rest we will have to change later. Thanks!

**Review by PotatoCombat — APPROVED** [review]
Strange I thought they were refactored correctly. Thanks for this!
